cdap-common/bin/service

```diff
@@ -103,6 +103,10 @@ _start_java() {
     cdap_set_hbase || exit 1
     # Master requires this local directory
     cdap_check_or_create_master_local_dir || die "Could not create Master local directory"
+    "${JAVA}" "${JAVA_HEAPMAX}" -Dhive.classpath=${HIVE_CLASSPATH} ${OPTS} -cp ${CLASSPATH} co.cask.cdap.master.startup.MasterStartupTool </dev/null >>${loglog} 2>&1
```

Comment:
``HIVE_CLASSPATH`` or ``EXPLORE_CLASSPATH`` ?  and do you not need ``-Dexplore.conf.files`` ?